app/src/main/java/com/parasoft/demoapp/component/OrderDialog.java

```diff
@@ -191,6 +192,7 @@ public void onResponse(@NonNull Call<ResultResponse<OrderResponse>> call, @NonNu
                     int code = response.code();
                     if (code == 200) {
                         orderInfo = response.body().getData();
+                        orderInfo.setSubmissionDate(TimeUtil.convertToLocalTime(orderInfo.getSubmissionDate()));
```

Comment:
Instead of modifying the "orderInfo" object, please move the util call inside "OrderDialog.setOrderLayout" to get a string of converted time right before setting text fields.